helm/linkerd2-app/templates/psp.yaml

```diff
@@ -84,6 +84,11 @@ roleRef:
   name: linkerd-psp
   apiGroup: rbac.authorization.k8s.io
 subjects:
+{{ if .Values.tracing.enabled -}}
+- kind: ServiceAccount
+  name: linkerd-collector
+  namespace: {{.Release.Namespace}}
+{{ end -}}
```

Comment:
although the subcharts are removed, I've retained these for ease of syncing with upstream